cmake/modules/BuildPlugin.cmake

```diff
@@ -56,6 +56,7 @@ MACRO(BUILD_PLUGIN PLUGIN_NAME)
 
 	ADD_LIBRARY(${PLUGIN_NAME} ${PLUGIN_LINK} ${PLUGIN_SOURCES} ${plugin_MOC_out} ${RCC_OUT})
 
+	set_target_properties(${PLUGIN_NAME} PROPERTIES AUTOUIC_SEARCH_PATHS ${PLUGIN_DIR})
```

Comment:
Is this line still needed? As I mentioned in the other thread, the plugin UI files are alongside the files that reference them, so CMake should be able to find them without additional hints. Even if it can't, this property is better specified in the individual plugins' `CMakeLists.txt` files, so we don't assume a particular plugin file layout in this generic function.

Reply:
> Is this line still needed?

without this line, the build fails. I have moved it to the individual plugins now. Let's wait and see.

Reply:
Looks like it got fixed.